Lecture2/Task4/com/playtika/ArrayAndLinkedLists.java

```diff
@@ -0,0 +1,110 @@
+        System.out.println("Time delta for created ArrayList: " + deltaForDeleteItemsArrayList);
+
+        long deltaForDeleteitemsLinkedList = measureForDeletedElementInLinkedList(createdLinkedList);
+        System.out.println("Time delta for delete 1000 elements from LinkedList:" + deltaForDeleteitemsLinkedList);
+
+
+        long deltaForFoundedItemsArrayList = measureForFoundedValuesInArrayList(createdArrayList);
+        System.out.println("Time delta for count 1000 elements from ArrayList: " + deltaForFoundedItemsArrayList);
+
+
+        long deltaForFoundedElementsLinkedList = measureForFoundedValuesInLinkedList(createdLinkedList);
+        System.out.println("Time delta for count 1000 elements from LinkedList: " + deltaForFoundedElementsLinkedList);
+    }
+
+    private static void addRandomToCreatedList(List<Integer> integers) {
+        for (int i = 0; i < 1000000; i++) {
+            int a = RANDOM.nextInt(1000);
+            integers.add(a);
+        }
+
+    }
+
+    private static long addRandomtoCreatedArrayListMeasure(List<Integer> createdArrayList) {
+        long startTime = System.nanoTime();
+        addRandomToCreatedList(createdArrayList);
+        long endTime = System.nanoTime();
+        return (endTime - startTime);
+    }
+
+
+    private static long addRandomCreatedLinkedListMeasure(List<Integer> createdLinkedList) {
+
+        long startTime = System.nanoTime();
+        addRandomToCreatedList(createdLinkedList);
+        long endTime = System.nanoTime();
+        return (endTime - startTime);
+    }
+
+
+    private static void deleteElementsFormSelectedList(List<Integer> integers) {
+
+        int i = 0;
+        while (i < 1000) {
+            integers.remove(i);
+            i++;
+        }
+
+    }
+
+    private static long measureForDeletedElementInLinkedList(List<Integer> createdLinkedList) {
+        long startTime = System.nanoTime();
+        deleteElementsFormSelectedList(createdLinkedList);
+        long endTime = System.nanoTime();
+        return (endTime - startTime);
+    }
+
+    private static long measureForDeletedElementForArrayList(List<Integer> createdArrayList) {
+        long startTime = System.nanoTime();
+        deleteElementsFormSelectedList(createdArrayList);
+        long endTime = System.nanoTime();
+        return (endTime - startTime);
+
+    }
+
+    private static void foundByElement(List<Integer> integers) {
+
+        int i = 0;
+        while (i < 1000) {
+            integers.contains(i);
+            i++;
+        }
+    }
+
+    private static long measureForFoundedValuesInArrayList(List<Integer> createdArrayList) {
+        long startTime = System.nanoTime();
+        foundByElement(createdArrayList);
+        long endTime = System.nanoTime();
+        return (endTime - startTime);
+    }
+
+    private static long measureForFoundedValuesInLinkedList(List<Integer> createdLinkedList) {
```

Comment:
what is a difference between `measureForFoundedValuesInLinkedList ` and `measureForFoundedValuesInArrayList `?